core/Network/TLS/Extra/Cipher.hs

```diff
@@ -45,6 +45,9 @@ module Network.TLS.Extra.Cipher
     , cipher_ECDHE_ECDSA_AES256CBC_SHA384
     , cipher_ECDHE_ECDSA_AES128GCM_SHA256
     , cipher_ECDHE_ECDSA_AES256GCM_SHA384
+    -- * TLS 1.3
+    , cipher_AES128GCM_SHA256
+    , cipher_AES256GCM_SHA384
```

Comment:
Therefore, I'd suggest constructing new names for these.  Perhaps:
```
, cipher_TLS_AES_128_GCM_SHA256
, cipher_TLS_AES_256_GCM_SHA384
```
matching the names in spec verbatim (with a `cipher_` prefix).